documentation/STIR-UsersGuide.tex

```diff
@@ -3557,6 +3557,39 @@ \subsubsection{
 in the input image (in that direction).
 \end{description}
 
+{ \subsubsubsection{Separable Gaussian Filter}
+}
+This is a separable 3D Gaussian filter for images discretised on
+a Cartesian voxel grid. These filters are composed of 3 1D filter
+kernels, one for the x,y and z directions each. Each 1D filter
+is specified by a Full Width at Half Maximum (FWHM) parameter
+(in millimetres), and a maximum spatial kernel width.
+The kernel values are obtained by sampling a Gaussian.
+The rules which apply to the construction and application of the Gaussian filter follow the one of the Metz filter.
```

Comment:
```suggestion
The rules which apply to the construction and application of the Gaussian filter follow the one of the Metz filter.
However, that filter is based on sampling the kernel in frequency space. It can therefore have negative
values, even for power $0$, in contrast to the current filter.
```